### PR TITLE
Add support for Gaomon M1230

### PR DESCRIPTION
**Pull request by jamesbt365**
Verification: https://discord.com/channels/615607687467761684/932080927381782558/932100560105132032

Button count may need to be 12.

### CODE REVIEW

File: OpenTabletDriver.Configurations/Configurations/Gaomon/M1230.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      "MaxY": 34305.0
+    },
+    "Pen": {
+      "MaxPressure": 8191,
+      "Buttons": {
+        "ButtonCount": 2
+      }
+    },
+    "AuxiliaryButtons": {
+      "ButtonCount": 13
```
Comment (X9VoiD) [inline OpenTabletDriver.Configurations/Configurations/Gaomon/M1230.json:17]: Why 13? Current parsers only handle up to 12 buttons.

**Review by X9VoiD — CHANGES_REQUESTED** [review]

File: OpenTabletDriver.Configurations/Configurations/Gaomon/M1230.json (hunk 1 above)
Comment (jamesbt365) [reply OpenTabletDriver.Configurations/Configurations/Gaomon/M1230.json:17]: Good point, i'll correct this, just thought a button might be dropped off but i'm not too sure.

File: TABLETS.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 | ---------------------------   | :---------------: | ----- |
 | Acepen AP 1060                |     Supported     |
 | Gaomon 1060 Pro               |     Supported     |
+| Gaomon M1230                  |     Supported     |
```
Comment (X9VoiD) [inline TABLETS.md:5]: Gaomon M1230 has other functions that OTD does not support yet.

Comment (X9VoiD) [reply TABLETS.md:5]: Touch bar is not supported yet.

**Comment by InfinityGhost** [comment]
Resolve merge conflicts and I'll merge.

**Comment by jamesbt365** [comment]
Merge conflicts resolved.

**Review by InfinityGhost — APPROVED** [review]

**Pull request merged**